src/Tags/Tagging.hs

```diff
@@ -29,30 +30,41 @@ runTagging blob symbolsToSummarize
   = Eff.run
   . evalState @[ContextToken] []
   . Streaming.toList_
-  . contextualizing blob symbolsToSummarize
+  . contextualizing blob toKind
   . tagging blob
+  where
+    toKind x = do
+      guard (x `elem` symbolsToSummarize)
+      case x of
+        "Function" -> Just Function
+        "Method"   -> Just Method
+        "Class"    -> Just Class
+        "Module"   -> Just Module
+        "Call"     -> Just Call
+        "Send"     -> Just Call
+        _          -> Nothing
```

Comment:
In addition to filtering, we also translate the string coming from `ConstructorName` into a well-formed `Kind`.